x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -158,13 +159,31 @@ async function resetCredentials(
         },
       }
     );
-    return {
-      password: response.data.password,
-      username: response.data.username,
-    };
-  } catch (error) {
-    throw new Error(`${error.message}`);
-  }
+    if (response.status !== 200) {
+      throw new Error('Credentials have not been reset. Retrying in 20s...');
+    } else {
+      log.info('Credentials have ben reset');
+      return {
+        password: response.data.password,
+        username: response.data.username,
+      };
+    }
+  };
+
+  const retryOptions = {
+    onFailedAttempt: (error: Error | AxiosError) => {
+      if (error instanceof AxiosError && error.code === 'ENOTFOUND') {
+        log.info('Project is not reachable. Retrying in 20s...');
```

Comment:
nit: `p-retry` uses `retry` which provided exponential fallback. In the other words first retries happen in less than 20 seconds. The formulae is specified [here](https://www.npmjs.com/package/retry) which is

```
retryDelay = Math.min(random * minTimeout * Math.pow(factor, attempt), maxTimeout)
```

we just limit the maximum delay to 20 seconds via `maxTimeout`.